requirements.txt

```diff
@@ -1,6 +1,6 @@
 koji
 rpm-py-installer
 requests
-PyYAML
+PyYAML==5.4.1
```

Comment:
This most likely should not be pinned here.  If it has to be pinned for CI to pass, it should be in some file which is only applied during CI.

The problem with adding this here is that this requirements.txt ends up being used in the package metadata.  I think it means you won't be able to install alt-src into a virtualenv along with any other package which also pins a different version of PyYAML.  This would probably also prevent alt-src command from running when installed from an RPM, if the system version of PyYAML doesn't exactly match this.

In summary, if PyYAML has to be pinned that should probably be done via test-requirements.txt or legacy.constraints right now.

However, can you share a bit more info about why you wanted to do this first?  e.g. can you point at a test failure which is being fixed by this?  It might be easier and better to just do a proper fix in the code...